clarify priority in user demand node (#1746)
Deltares/Ribasim: docs/reference/node/user-demand.qmd

With this the demand can be updated over time. In between the given times the
demand is interpolated linearly, and outside the demand is constant given by the
nearest time value.

> And `priority` parameter is not dependent on time and remains an integer value for each node.

Reviewer: ```suggestion
The `priority` is not allowed to change over time.
```
